airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/AbstractSshPostgresSourceAcceptanceTest.java

```diff
@@ -24,45 +24,79 @@
 
 package io.airbyte.integrations.io.airbyte.integration_tests.sources;
 
+import static io.airbyte.integrations.base.ssh.SshBastion.*;
```

Comment:
style: prefer using explicit import instead of star imports

Reply:
> style: prefer using explicit import instead of star imports

replaced star import with explicit